### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by aparajita**
**Summary**
This PR fixes some grammatical errors in the docs.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot: 

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge
- [x] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ Function for transforming header texts into slugs. This affects the ids/links ge
 - Type: `Object`
 - Default: `{ target: '_blank', rel: 'noopener noreferrer' }`
 
-The key and value pair will be added to `<a>` tags that points to an external link. The default option will open external links in a new window.
+A key and value pair that will be added to `<a>` tags that point to an external link. The default option will open external links in a new window.
```
Comment (ulivz) [inline docs/config/README.md]: 2 `that` here.

Comment (aparajita) [reply docs/config/README.md]: Not sure what you mean...

Comment (ulivz) [reply docs/config/README.md]: You added a that, causing this sentence to appear two that, it is not fluent...

**Comment by aparajita** [comment]
See #819

**Pull request closed without merging**